### PR TITLE
Fixed broken CI and some minor bug fixes for ecommerce credentials due to test credential overwrite

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**
- Fixed CI for CommercialInvoice caused by improperly PriceSpeficiation
- Updated other credentials because chargeSpecification schema was removed and switched to PriceSpecification (reused chargeSpecification schema and tests)
- Fixed a naming bug for credential tests for 3 ecommerce credentials
- removed a couple of warnings because of the PriceSpecification issue
- no more warnings for any e-commerce schemas and their tests

### CODE REVIEW

**Comment by Therecanbeonlyone1969** [comment]
CI & CD passed .... https://github.com/cstoecker/traceability-vocab/actions/runs/616336361 ... https://github.com/cstoecker/traceability-vocab/actions/runs/616336359

@OR13 @mprorock @mkhraisha

**Comment by mprorock** [comment]
@OR13 merge order between this and #127 ?

**Comment by mprorock** [comment]
@Therecanbeonlyone1969 will approve once we sort merge order on the CD/CI refactor

**Comment by OR13** [comment]
both these PRs are big, so it will suck either way.

There are 2 ways to think of merge order.

- Smaller PRs first.

- First to passing CI test wins.

If we merge this first, I will likely need to reimplement it in my PR.

I can probably handle the merge in my PR.

**Review by OR13 — CHANGES_REQUESTED** [review]
@Therecanbeonlyone1969 can you remove the auto generated changes from this PR? I will then reimplement the test / code changes you have made in https://github.com/w3c-ccg/traceability-vocab/pull/127

Just overwrite `./docs/`  (except for schemas folder) with origin main docs and I will handle the merge in my PR

**Comment by Therecanbeonlyone1969** [comment]
> @Therecanbeonlyone1969 can you remove the auto generated changes from this PR? I will then reimplement the test / code changes you have made in #127
> 
> Just overwrite `./docs/` (except for schemas folder) with origin main docs and I will handle the merge in my PR

@OR13 ... ok will do that. Gimmer a hot minute.

**Comment by Therecanbeonlyone1969** [comment]
@OR13 ... done

File: docs/schemas/EcommerceWayBillTotals.json [hunk 1]
```diff
@@ -55,7 +55,7 @@
         "totalRateCharge": {
             "$comment": "{\"term\": \"totalRateCharge\", \"@id\": \"https://schema.org/totalPrice\"}",
             "title": "rateCharge",
-            "$ref": "https://w3id.org/traceability/schemas/chargeSpecification.json"
+            "$ref": "https://w3id.org/traceability/schemas/PriceSpecification.json"
```
Comment (OR13) [inline docs/schemas/EcommerceWayBillTotals.json:58]: ✅

File: docs/schemas/IssuerAgent.json [hunk 2]
```diff
@@ -59,12 +59,12 @@
         "charge": {
             "$comment": "{\"term\": \"charge\", \"@id\": \"https://schema.org/Price\"}",
             "title": "Charge",
-            "$ref": "https://w3id.org/traceability/schemas/chargeSpecification.json"
+            "$ref": "https://w3id.org/traceability/schemas/PriceSpecification.json"
```
Comment (OR13) [inline docs/schemas/IssuerAgent.json:62]: ✅

File: docs/schemas/IssuerAgent.json [hunk 3]
```diff
@@ -59,12 +59,12 @@
...
         },
         "declaredValueCarriage": {
             "$comment": "{\"term\": \"declaredValueCarriage\", \"@id\": \"https://schema.org/totalPrice\"}",
             "title": "declaredValueCarriage",
-            "$ref": "https://w3id.org/traceability/schemas/chargeSpecification.json"
+            "$ref": "https://w3id.org/traceability/schemas/PriceSpecification.json"
```
Comment (OR13) [inline docs/schemas/IssuerAgent.json:67]: ✅

File: docs/schemas/IssuerAgent.json [hunk 4]
```diff
@@ -78,7 +78,7 @@
         "declaredValueCustoms": {
             "$comment": "{\"term\": \"declaredValueCustoms\", \"@id\": \"https://schema.org/discountPrice\"}",
             "title": "declaredValueCustoms",
-            "$ref": "https://w3id.org/traceability/schemas/chargeSpecification.json"
+            "$ref": "https://w3id.org/traceability/schemas/PriceSpecification.json"
```
Comment (OR13) [inline docs/schemas/IssuerAgent.json:81]: ✅

File: docs/schemas/IssuerAgent.json [hunk 5]
```diff
@@ -92,7 +92,7 @@
         "amountInsurance": {
             "$comment": "{\"term\": \"amountInsurance\", \"@id\": \"https://schema.org/PaymentChargeSpecification\"}",
             "title": "amountInsurance",
-            "$ref": "https://w3id.org/traceability/schemas/chargeSpecification.json"
+            "$ref": "https://w3id.org/traceability/schemas/PriceSpecification.json"
```
Comment (OR13) [inline docs/schemas/IssuerAgent.json:95]: ✅

File: docs/schemas/PriceSpecification.json [hunk 6]
```diff
@@ -23,7 +23,7 @@
       "$comment": "{\"term\": \"price\", \"@id\": \"https://schema.org/price\"}",
       "title": "Price",
       "description": "The offer price of a product, or of a price component when attached to PriceSpecification and its subtypes.",
-      "type": "string"
+      "type": "number"
```
Comment (OR13) [inline docs/schemas/PriceSpecification.json:26]: ✅

File: docs/schemas/ShippingStop.json [hunk 7]
```diff
@@ -8,8 +8,6 @@
     "required": [
         "shippingStopAddress",
         "carrier",
-        "vesselNumber",
```
Comment (OR13) [inline docs/schemas/ShippingStop.json:11]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json [hunk 8]
```diff
@@ -65,7 +65,7 @@
     },
     "totalPaymentDue": {
       "type": "PriceSpecification",
-      "price": "150000.00",
+      "price": 150000.00,
```
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:68]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json [hunk 9]
```diff
@@ -83,7 +83,7 @@
         "quantity": "5",
         "priceSpecification": {
           "type": "PriceSpecification",
-          "price": "10000.00",
+          "price": 10000.00,
```
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:86]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json [hunk 10]
```diff
@@ -101,7 +101,7 @@
         "quantity": "10",
         "priceSpecification": {
           "type": "PriceSpecification",
-          "price": "10000.00",
+          "price": 10000.00,
```
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:104]: ✅

File: packages/generate-examples/src/__tests__/EcommerceBindingDataRegistrationCredential.test.js [hunk 11]
```diff
@@ -56,7 +56,7 @@ describe("Generate", () => {
     fs.writeFileSync(
       path.resolve(
         __dirname,
-        "../../../../docs/credentials/EcommercePackageRegistrationCredential.json"
+        "../../../../docs/credentials/EcommerceBindingDataRegistrationCredential.json"
```
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceBindingDataRegistrationCredential.test.js:59]: ✅

File: packages/generate-examples/src/__tests__/EcommerceProductRegistrationCredential.test.js [hunk 12]
```diff
@@ -24,7 +24,10 @@ describe("Generate", () => {
         credentialSubject: {
           id: "did:web:www.acme.com",
           type: "EcommerceProductRegistrationCredential",
-          productCode: faker.random.number(14),
+          productCode: faker.random.number({
```
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceProductRegistrationCredential.test.js:27]: ✅

File: packages/generate-examples/src/__tests__/EcommerceWayBillRegistrationCredential.test.js [hunk 13]
```diff
@@ -57,7 +57,7 @@ describe("Generate", () => {
     fs.writeFileSync(
       path.resolve(
         __dirname,
-        "../../../../docs/credentials/EcommercePackageRegistrationCredential.json"
+        "../../../../docs/credentials/EcommerceWayBillRegistrationCredential.json"
```
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceWayBillRegistrationCredential.test.js:60]: ✅

File: packages/traceability-vocab/src/generators/EcommerceBindingDataRegistrationCredential.js [hunk 14]
```diff
@@ -3,21 +3,25 @@ const { getOrganization } = require('./Organization');
 const { getShippingStop } = require('./ShippingStop');
 
 const getEcommerceBindingDataRegistrationCredential = () => {
-  const finalVesselID = `ACMEVessel#${faker.random.number({ min: 1, max: 999 })}`;
 
-  const CarrierName = getOrganization();
-  const finalCarrierName = CarrierName.name;
+    const finalVesselID = `ACMEVessel#${faker.random.number({ min: 1, max: 999 })}`;
```
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceBindingDataRegistrationCredential.js:7]: ✅

File: packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js [hunk 15]
```diff
@@ -1,5 +1,5 @@
 const faker = require('faker');
-const { getchargeSpecification } = require('./chargeSpecification');
+const { getPriceSpecification } = require('./PriceSpecification');
```
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js:2]: ✅

File: packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js [hunk 16]
```diff
@@ -13,7 +13,7 @@ const getEcommerceWayBillTotals = () => {
     return faker.random.arrayElement(types);
   };
 
-  const totalRateCharge = getchargeSpecification();
+  const totalRateCharge = getPriceSpecification();
```
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js:16]: ✅

File: packages/traceability-vocab/src/generators/IssuerAgent.js [hunk 17]
```diff
@@ -1,6 +1,6 @@
 const faker = require('faker');
 const { getOrganization } = require('./Organization');
-const { getchargeSpecification } = require('./chargeSpecification');
+const { getPriceSpecification } = require('./PriceSpecification');
```
Comment (OR13) [inline packages/traceability-vocab/src/generators/IssuerAgent.js:3]: ✅

File: packages/traceability-vocab/src/generators/IssuerAgent.js [hunk 18]
```diff
@@ -32,12 +32,12 @@ const getIssuerAgent = () => {
     iataCode,
     accountNumber,
     accountingInformation: getDescription(),
-    charge: getchargeSpecification(),
-    declaredValueCarriage: getchargeSpecification(),
+    charge: getPriceSpecification(),
```
Comment (OR13) [inline packages/traceability-vocab/src/generators/IssuerAgent.js:35]: ✅

**Comment by OR13** [comment]
I have caught up with all these changes and am closing this PR.

**Pull request closed without merging**